src/deck-options.md

```diff
@@ -386,193 +344,149 @@ The options are:
   one of your clients doesn't support it, things will not work correctly.
 - If you previously used the 'custom scheduling' version of FSRS, please make
   sure you clear out the custom scheduling section before enabling FSRS.
 
-#### FSRS Options
-
-**Desired Retention**
+### Desired Retention
 
-Desired retention controls how likely you are to remember cards when they are reviewed.
-The default value of 0.9 will schedule cards so you have a 90% chance of remembering
-them when they come up for review again.
+Desired retention controls how likely you are to remember cards when they are scheduled for a review.
+The default value of 0.90 will schedule cards so you have a 90% chance of remembering
+them when they come up for review again. This should normally translate to remembering around 90% cards when they are reviewed, and only failing around 10%.
 
 Here is a graph that shows how adjusting this value will affect your workload:
 
-<img src="media/FSRS_retention.png" width="600">
+![graph showing an exponential increase in workload as desired retention nears one.](media/FSRS_retention.png)
 
-There are two things to notice:
+The exact shape of the graph is different for everyone. However, there are two patterns that hold true for all:
 
-- As desired retention approaches 1.0, the frequency that you need to review cards
-  increases drastically. For example, imagine you have a card that you have a 90%
-  chance of remembering after 100 days. If your desired retention was 0.95, you'd
-  need to review it after 47 days instead (approximately twice as frequently).
-  At 0.97, the delay would be only 27 days (approximately 3.7x as frequently).
-  At 0.99, you'd be reviewing every 9 days (more than 10x what you'd be doing with
-  the defaults).
+- As desired retention approaches 1.0, the workload increases drastically.
+  Imagine you have a card with a 90% chance of remembering it after 100 days. If your desired retention is 0.90, you'll review the card again in 100 days. But if your desired retention is 0.95, you'll need to review it after 46 days instead.
+  This means that the intervals of your cards almost halve at 0.95 desired retention and you need to review cards twice as frequently compared to 0.90 desired retention.
+  At 0.97, the interval will be 27 days (you'll have to review your cards 3.7x as frequently).
+  At 0.99, the interval will be only 9 days (you'll have to review your cards more than 10x more frequently than with the defaults).
 
 - As desired retention decreases, you'll forget a greater percentage of your
   cards, and those cards will need to be reviewed again. Eventually, you'll
   get to a point where the forgotten cards contribute more to your workload
   than you gain from the longer delays, which is why you see the workload
-  on the left of the graph increasing. Also, bear in mind that forgetting
+  on the left of the graph increasing. Also, note that forgetting
```

Comment:
'Note' feels a bit strange there. How about "keep in mind" or "don't forget"?

Reply:
```suggestion
  on the left of the graph increasing. Also, keep in mind that forgetting
```